js/src/dapps/registry/Lookup/lookup.js

```diff
@@ -86,14 +102,26 @@ export default class Lookup extends Component {
     );
   }
 
-  onNameChange = (e) => {
-    this.setState({ name: e.target.value });
+  onInputChange = (e) => {
+    this.setState({ input: e.target.value });
```

Comment:
This function could do with a blank space at the end of it before the next starts.